code/modules/clothing/suits/sweater_gags.dm

```diff
@@ -0,0 +1,23 @@
+/obj/item/clothing/suit/sweater_gags
```

Comment:
You don't need to specify gags, especially not in the typepath. It's just a tool to use on things not a unique trait of the thing